packages/gestalt/src/Pog.css

```diff
@@ -163,20 +163,28 @@
 }
 
 .transparentDarkGray {
-  background-color: var(--color-background-button-semitransparentdark-default);
+  background-color: var(
```

Comment:
why Pog?

Reply:
The changes in Pog/IconButton are to prevent the Button token from affecting it, which would break it (since IconButton transparentDarkGray would go from near full black to near full white on hover, making it impossible to get good contrast unless the iconcolor would change)
This is how transparentDarkGray would look on hover without this change:
![Screenshot 2024-10-02 at 07 03 22](https://github.com/user-attachments/assets/01a235bb-f31e-467b-8f61-ae77e8fb1d84)